webpack.config.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable */
```

Comment:
- **[OPTIONAL]** Please do not disable linters ❌  it will help you to maintain better practices of JS, please 🙏🏽 correct ✔️ the linter errors manually.